plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -0,0 +1,157 @@
+package io.jenkins.plugins.casc;
+
+import java.util.*;
+import io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class SchemaGeneration {
+
+    public static String generateSchema() {
+
+        /**
+         * The initial template for the JSON Schema
+         */
+        StringBuilder schemaString = new StringBuilder();
+        schemaString.append("{\n" +
```

Comment:
this should either be in a constant or a resource file

Reply:
Yeah I will add a file called schema there :)

Reply:
@casz I could save the resources path in a static string and then save the JSONSchema in the resources folder .That could be picked up from the test file as well.What do you think ? 
Since using `getClass().getResource("/")` is proving  a bit of a pain :disappointed:

Reply:
Make it a constant?

Reply:
Rather we would anyway need to store the schema in a resource folder.What do you think of writing the schema into a `Schema.json` in a resource folder.

Reply:
`public static String resourcePath = "./src/main/resources/Schema.json";`

Reply:
The schema will be different for each Jenkins master, so we will never store a schema.json

Reply:
But for tests wont we need the schema to validate against a yaml file ? Wont we need to have a schema to validate against.

Reply:
We will generate schema in a given test and then we could have our test validate against all of our existing .yml files in the resources folder yes.

Reply:
Again mixing things here. @timja initial comment was that the very static
```
{
"$schema": "http://json-schema.org/draft-06/schema#",
"id": "http://jenkins.io/configuration-as-code#",
"description": "Jenkins Configuration as Code",
"type": "object"
}
```
Should be kept in a constant or a file. For simplicity, I suggest it is stored as a constant of type JSONObject.

Reply:
Yeah @casz I got that :) 
I just wanted your feedback on storing it in the test hence the different question.